hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/ConfigProviderImpl.java

```diff
@@ -40,30 +91,89 @@ public class ConfigProviderImpl implements ConfigProvider {
+                .withSource(dynamicConfigSource)
+                .withConverter(new CongestionMultipliersConverter())
+                .withConverter(new EntityScaleFactorsConverter())
+                .withConverter(new EntityTypeConverter())
+                .withConverter(new KnownBlockValuesConverter())
+                .withConverter(new LegacyContractIdActivationsConverter())
+                .withConverter(new MapAccessTypeConverter())
+                .withConverter(new RecomputeTypeConverter())
+                .withConverter(new ScaleFactorConverter())
+                .withConverter(new AccountIDConverter())
+                .withConverter(new ContractIDConverter())
+                .withConverter(new FileIDConverter())
+                .withConverter(new HederaFunctionalityConverter())
+                .withConverter(new ProfileConverter())
+                .withConverter(new SidecarTypeConverter())
+                .withConverter(new KeyValuePairConverter())
+                .withValidator(new EmulatesMapValidator())
+                .withConfigDataType(AccountsConfig.class)
+                .withConfigDataType(AutoCreationConfig.class)
+                .withConfigDataType(AutoRenew2Config.class)
+                .withConfigDataType(AutoRenewConfig.class)
+                .withConfigDataType(BalancesConfig.class)
+                .withConfigDataType(BootstrapConfig.class)
+                .withConfigDataType(CacheConfig.class)
+                .withConfigDataType(ConsensusConfig.class)
+                .withConfigDataType(ContractsConfig.class)
+                .withConfigDataType(CryptoCreateWithAliasConfig.class)
+                .withConfigDataType(DevConfig.class)
+                .withConfigDataType(EntitiesConfig.class)
+                .withConfigDataType(ExpiryConfig.class)
+                .withConfigDataType(FeesConfig.class)
+                .withConfigDataType(FilesConfig.class)
+                .withConfigDataType(GrpcConfig.class)
+                .withConfigDataType(HederaConfig.class)
+                .withConfigDataType(IssConfig.class)
+                .withConfigDataType(LazyCreationConfig.class)
+                .withConfigDataType(LedgerConfig.class)
+                .withConfigDataType(NettyConfig.class)
+                .withConfigDataType(QueriesConfig.class)
+                .withConfigDataType(RatesConfig.class)
+                .withConfigDataType(SchedulingConfig.class)
+                .withConfigDataType(SigsConfig.class)
+                .withConfigDataType(StakingConfig.class)
+                .withConfigDataType(StatsConfig.class)
+                .withConfigDataType(TokensConfig.class)
+                .withConfigDataType(TopicsConfig.class)
+                .withConfigDataType(TraceabilityConfig.class)
+                .withConfigDataType(UpgradeConfig.class)
+                .withConfigDataType(UtilPrngConfig.class)
+                .withConfigDataType(VirtualdatasourceConfig.class)
```

Comment:
nit : 
```suggestion
                .withConfigDataType(VirtualDataSourceConfig.class)
```

Reply:
That won't compile. I named all the config record the same pattern (prefix of the property name). But we can change it in a next PR that structures the properties in a better way.